src/entities/SurveyTopic/utils.ts

```diff
@@ -0,0 +1,23 @@
+import { ReactionType, Survey } from './types'
+
+type ReactionView = { reaction: ReactionType; icon: string }
```

Comment:
Type instead of reaction as this is a list of reactions? 
`{ type: ReactionType; icon: string }`